README.md

```diff
@@ -25,6 +25,20 @@ In addition to that, Minigalaxy also allows you to:
 - Use the system's ScummVM or DOSBox installation
 - Install Windows games using Wine
 
+### Backwards compatibility
+Minigalaxy version 1.3.2 and higher change some aspects of windows game installations through wine.
+It will try to adapt already installed games to the new concept when launched through Minigalaxy.
+
+However, this will *likely break games* that save some directory paths in the *windows registry*.
+In that case, only a reinstallation will repair the game. 
```

Comment:
I don't think want this. We should just use the old behavior for old game installations.

Reply:
Lets just make sure the linking works in general by testing it well.

I'll ask you to remove this second from the README, since Minigalaxy has never worked with registry key installation through the installer, since it would move the files afterwards.